backend/src/libs/packages/s3/s3.package.ts

```diff
@@ -90,6 +94,17 @@ class S3 {
     );
   }
 
+  public getFileKey(url: string | null): string | null {
+    if (!url) {
+      return null;
+    }
+
+    const match = url.match(/https:\/\/[^/]+\/([^/]+)/);
+    const fileKeyIndex = 1;
```

Comment:
According to the quality criteria rule B22 constants should be named using UPPER_CASE. And let's declare it outside of this method, ideally in separate file